machida/cpp/python-wallaroo.c

```diff
@@ -238,7 +235,9 @@ extern long partition_function_partition_u64(PyObject *partition_function, PyObj
   Py_DECREF(pFunc);
 
   long rtn = PyInt_AsLong(pValue);
```

Comment:
Looking at the documentation for this (https://docs.python.org/2/c-api/int.html#c.PyInt_AsLong) I think it will return `-1` if `pValue` is null. I don't think that's exactly what we want to do. That said, we're getting rid u64 key entirely when we switch to consistent hashing, so this won't be an issue for long.